packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -59,38 +56,43 @@ import {updateBreakpoints} from '../../utils/replace-breakpoint';
 })
 export class AtomicResultList implements InitializableComponent {
   @InitializeBindings() public bindings!: Bindings;
-  private resultList!: ResultList;
+  public resultList!: ResultList;
   public resultsPerPage!: ResultsPerPage;
-  private resultTemplatesManager!: ResultTemplatesManager<TemplateContent>;
+  public listWrapperRef?: HTMLDivElement;
 
-  @Element() private host!: HTMLDivElement;
+  @Element() public host!: HTMLDivElement;
+
+  /**
+   * TODO: KIT-452 Infinite scroll feature
+   * Whether to automatically retrieve an additional page of results and append it to the
+   * current results when the user scrolls down to the bottom of element
+   */
+  private enableInfiniteScroll = false;
 
   @BindStateToController('resultList')
   @State()
-  private resultListState!: ResultListState;
+  public resultListState!: ResultListState;
 
   @BindStateToController('resultsPerPage')
   @State()
-  private resultsPerPageState!: ResultsPerPageState;
+  public resultsPerPageState!: ResultsPerPageState;
+
+  @State() public ready = false;
 
   @State() public error!: Error;
-  @State() private templateHasError = false;
+  @State() public templateHasError = false;
+
+  public resultListCommon!: ResultListCommon;
 
-  /**
-   * TODO: KIT-452 Infinite scroll feature
-   * Whether to automatically retrieve an additional page of results and append it to the
-   * current results when the user scrolls down to the bottom of element
-   */
-  private enableInfiniteScroll = false;
   /**
    * A list of non-default fields to include in the query results, separated by commas.
-   * The default fields sent in a request are: 'date', 'author', 'source', 'language', 'filetype', 'parents', ‘urihash’, ‘objecttype’, ‘collection’, ‘permanentid’ 'ec_price', 'ec_name', 'ec_description', 'ec_brand', 'ec_category', 'ec_item_group_id', 'ec_shortdesc', 'ec_thumbnails', 'ec_images', 'ec_promo_price', 'ec_in_stock', 'ec_cogs', and 'ec_rating'.
+   * TODO: add a note about headless?.
```

Comment:
This todo would appear on the website reference doc

Reply:
I wasn't planning on leaving it, I just wanted to know what I should do there instead!